### ISSUE TITLE
Invalid login url and url in email

### ISSUE THREAD

**Issue by Shnoulle** [YunoHost-Apps/friendica_ynh#85]
### Describe the bug

After installation : login url and url shown in email are bad ( no domain)

### Context

- Hardware: *Old omputer.*
- YunoHost version: 11.0.10.1
- I have access to my server: *Through SSH && through the webadmin && direct access via keyboard / screen*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*
- Using, or trying to install package version/branch: Friendica 2022.06~ynh1
- If upgrading, current package version: *can be found in the admin*

### Steps to reproduce

Via admin : 
- Set up a new domain
- Set up letsencrypt
- Set up all DNS 
- On this domain : setup friendica with a specific admin
- Open a private browser : open the domain link
- Log in with your selected admin user email
- Wait 
    - receive an email with `Localisation du site : http://` only
    - Get a 504 on browser
    - Reload : try to connect : redirected to https://login/
    - reload : check source : it's the action url

### Expected behavior

https:// must be https://friendica.example.org

![Capture d’écran du 2022-12-02 08-55-35](https://user-images.githubusercontent.com/1439428/205244942-13a31b77-faab-4d48-99ba-f95d7843d141.png)

Partial mail content

> Cher Shnoulle, 
> Merci pour votre inscription sur Friendica Social Network. 
> Votre compte a été créé.
> Les informations de connexion sont les suivantes :
> 
> Localisation du site :	http://
> Nom d'utilisateur :		shnoulle
> Mot de passe :		random1234
> 
> Vous pouvez changer votre mot de passe depuis la page "Paramètres" de votre compte après vous êtreconnecté.

**Comment by Shnoulle** [comment]
Via CLI

```
# yunohost app install friendica -a "domain=friendica.example.net"
Choose an administrator user for this app [shnoulle]: shnoulle
Choose the application language [de | en | es | fr | it | pt]: fr
Info: Installing friendica...
Info: [....................] > Validating installation parameters...
Info: [+...................] > Storing installation settings...
Info: [#+++++..............] > Installing dependencies...
Info: [######+.............] > Configuring system user...
Info: [#######+............] > Creating a MySQL database...
Info: [########++++........] > Setting up source files...
Info: [############+.......] > Configuring NGINX web server...
Info: [#############+......] > Adding a configuration file...
Info: [##############+++...] > Configuring PHP_FPM...
Info: [#################+..] > Setuping a cron job...
Info: [##################+.] > Reloading NGINX web server...
Info: [####################] > Installation of friendica completed
Success! Installation completed
```

Seems OK.

**Comment by mediaformat** [comment]
Fresh install `2023.01~ynh1`, getting the same issue. @Shnoulle Did installing via CLI fix the problem?

**Comment by mediaformat** [comment]
I've deactivated ipv6, uninstalled and reinstalled but the login form still goes to `https:///login`

**Comment by OniriCorpe** [comment]
hi!

this should be fixed with 2023.12~ynh2
